GUI/CalibrateTaskWindow.xaml.cs

```diff
@@ -11,6 +11,8 @@
 using TaskLayer;
 using Proteomics.ProteolyticDigestion;
 using MassSpectrometry;
+using System.Windows.Threading;
+using MetaMorpheusGUI.Util;
```

Comment:
remove `using MetaMorpheusGUI.Util;`
potentially remove the other using - what is this for?

Reply:
System.Windows.Threading is for the timer. should i just use `System.Windows.Threading.DispatcherTimer` ?

Reply:
no this is fine as is